src/test/java/org/apache/commons/crypto/stream/PositionedCryptoInputStreamTest.java

```diff
@@ -247,7 +250,7 @@ private void testSeekFailed(final String cipherClass, final int position, final
                 getCipher(cipherClass), bufferSize);
         try {
             in.seek(position);
-            Assert.fail("Excepted exception for cannot seek to negative offset.");
+            fail("Excepted exception for cannot seek to negative offset.");
         } catch (final IllegalArgumentException iae) {
```

Comment:
Hi @arturobernalg 
Thank you for your PR.
If we go to JUnit 5, then let's remove this pattern with `assertThrows()`
WDYT?

Reply:
Hi @garydgregory 
Agree with the approach.  Changed